packages/create-anywidget/index.js

```diff
@@ -22,56 +22,56 @@ ${grey(`create-anywidget version ${pkg.version}`)}
 p.intro("Welcome to anywidget!");
 
 if (cwd === ".") {
-	const dir = await p.text({
-		message: "Where should we create your project?",
-		placeholder: "  (hit Enter to use current directory)",
-	});
-	if (p.isCancel(dir)) process.exit(1);
-	if (dir) {
-		cwd = /** @type {string} */ (dir);
-	}
+  const dir = await p.text({
+    message: "Where should we create your project?",
+    placeholder: "  (hit Enter to use current directory)",
+  });
+  if (p.isCancel(dir)) process.exit(1);
+  if (dir) {
+    cwd = /** @type {string} */ (dir);
+  }
 }
 
 if (fs.existsSync(cwd)) {
-	if (fs.readdirSync(cwd).length > 0) {
-		let force = await p.confirm({
-			message: "Directory not empty. Continue?",
-			initialValue: false,
-		});
-		// bail if `force` is `false` or the user cancelled with Ctrl-C
-		if (force !== true) {
-			process.exit(1);
-		}
-	}
+  if (fs.readdirSync(cwd).length > 0) {
+    let force = await p.confirm({
+      message:
+        "Directory not empty. Continue? No files will be overwritten, and the git archive won't be touched.",
```

Comment:
Is this true? If I had `foo` and a already `src/foo/__init__.py` in my file system, wouldn't we blindly override?

Reply:
you're right, it will be overwritten!
I've removed that statement again.